### PR TITLE
Integration UI: bootstrap style, traceback, syntax highlighting

### PR DESCRIPTION
**Pull request by Piezoid**
- Uses bootstrap table and buttons,
- in debug mode, shows traceback when an exception is caught during integration,
- syntax highlighting on turtle preview (with pygments)

Screenshots :
- [Before (askomics/askomics/master)](http://i.imgur.com/GVbbTMl.png)
- [After (Piezoid/askomics/integration_ui)](http://i.imgur.com/xmkv99U.png)

### CODE REVIEW

**Comment by ofilangi** [comment]
+1

File: askomics/ask_view.py [hunk 1]
```diff
@@ -161,16 +166,39 @@ def preview_ttl(self):
...
+        {domain_knowledge_ttl}
+        """).format(header=sfc.get_turtle_template(),
+                    content_ttl = '\n'.join(src_file.get_turtle(preview_only=True)),
+                    abstraction_ttl = src_file.get_abstraction(),
+                    domain_knowledge_ttl = src_file.get_domain_knowledge()
+                    )
+
+        formatter = HtmlFormatter(cssclass='preview_field', nowrap=True, nobackground=True)
+        data = highlight(data, TurtleLexer(), formatter)
+        #FIXME: style is inlined in the <pre>, it should be added to the <head>
```
Comment (abretaud) [inline askomics/ask_view.py]: Can you do it in this PR too? (or is there something complicated in doing it?)

**Comment by abretaud** [comment]
It's much prettier, thanks a lot!

**Comment by abretaud** [comment]
+1 but if you can have a look at the fixme before merging

**Comment by abretaud** [comment]
Thanks!

**Pull request merged**